spring-cloud-stream/src/test/java/org/springframework/cloud/stream/aggregation/AggregationTest.java

```diff
@@ -443,4 +443,10 @@ else if (factory.getObjectType() == Processor.class) {
 	public static class DummyConfig {
 
 	}
+
+	@Configuration
+	@EnableAutoConfiguration
+	public static class FooConfig {
```

Comment:
Suspicious name for the configuration again. Even if it is test, you can't be sure that people who are going to maintain them after you would determine easily what this is about...

More over when you make yourself to think twice about names even in tests, you are going to be much better in naming on the production code level 😉 

Thanks